nuget/roundhouse/roundhouse.nuspec

```diff
@@ -12,6 +12,15 @@
     <licenseUrl>http://www.apache.org/licenses/LICENSE-2.0</licenseUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <tags>roundhouse db migration database migrator chucknorris</tags>
-	<iconUrl>https://raw.github.com/chucknorris/roundhouse/master/nuget/RoundhousE_Logo.NuGet.jpg</iconUrl>
+    <dependencies />
+	  <iconUrl>https://raw.github.com/chucknorris/roundhouse/master/nuget/RoundhousE_Logo.NuGet.jpg</iconUrl>
   </metadata>
+  <files>
+    <file src="$mergedExe$" target="bin\" />
+    <file src="$appConfigFile$" target="bin\" />
+    <file src="..\..\nuget\roundhouse\content\ReadMe.RoundhousE.txt" target="content\" />
+    <file src="..\..\docs\GettingStarted.docx" target="docs\" />

```

Comment:
What do you think about leaving this one out?

Reply:
By all means. I saw there was an issue that you should have left out some files to be Chocolatey "compatible" anyway. (#274).  I'll take out all but the ones mentioned in that one.